src/extensions/raw-output-example/pages/RawOutputExample.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <raw-content>
+    This page is using empty layout set in routes + no html template
+  </raw-content>
+</template>
+
+<script>
+export default {
+  name: 'RawOutputExample',
+  asyncData ({ store, route, context }) {
+    context.response.setHeader('Content-Type', 'text/xml')
```

Comment:
Here You can now set from the Vue component level server response headers + choose template